### PR TITLE
When adding task recognize PostgresPullQueue as PULL queue

### PR DESCRIPTION
**Pull request by jeanleonov**

### CODE REVIEW

File: AppTaskQueue/appscale/taskqueue/distributed_tq.py [hunk 1]
```diff
@@ -469,7 +470,7 @@ def __bulk_add(self, source_info, request, response):
       if (add_request.has_mode() and
           add_request.mode() == taskqueue_service_pb.TaskQueueMode.PULL):
         queue = self.get_queue(add_request.app_id(), add_request.queue_name())
-        if not isinstance(queue, PullQueue):
+        if not isinstance(queue, (PullQueue, PostgresPullQueue)):
```
Comment (sjones4) [inline AppTaskQueue/appscale/taskqueue/distributed_tq.py]: Perhaps not relevant for this pull request but there is a similar check in fetch_queue_stats that may also need updating.

Comment (jeanleonov) [reply AppTaskQueue/appscale/taskqueue/distributed_tq.py]: Thanks, I didn't notice that

**Comment by jeanleonov** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5301/

**Comment by jeanleonov** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/5337/

**Pull request merged**